src/libs/actions/Report.js

```diff
@@ -364,6 +355,19 @@ function updateReportWithNewAction(reportID, reportAction) {
         updatedReportObject.lastActorEmail = reportAction.actorEmail;
     }
 
+    // When handling an action from the current users we can assume that their
+    // last read actionID has been updated in the server but not necessarily reflected
+    // locally so we must first update it and then calculate the unread (which should be 0)
+    if (isFromCurrentUser) {
+        setLocalLastRead(reportID, newMaxSequenceNumber);
+    }
```

Comment:
This should not need to be moved from where it was before... unless I am missing something...

Reply:
Yeah Correct.

Reply:
Hmm ok this is kind of confusing and I think likely to break again when someone tries to clean up this logic.